cql/ManageRareAbnormality.cql

```diff
@@ -844,6 +948,40 @@ define SubsequentLowGradeCytologyReportsSinceInitialUnspecifiedHsilOrCin2InLast6
   else
     null
 
+
+define RecentNegativeHSILSurveillanceTests:
```

Comment:
`HSIL` -> `Hsil` for consistency. Same with `FirstTwoNegativeHSILSurveillanceTests` and `NegativeSurveillanceTestDatesAfterHSILSurveillance` expressions below.

Reply:
Agree - I'll make the names consistent.

Reply:
Names were updated in each of the places mentioned in my comment above, as well as areas where those expressions are referenced. Thanks.